From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Thu, 3 Mar 2016 19:18:27 +0100
Subject: [PATCH] bootstrap: Use public Ubuntu-ports mirrors.

Instead of relying on 127.0.0.1 partial Ubuntu mirror, use
public archives.
While doing so, switch to Ubuntu-ports, enabling  building arm64
bootstraps.

NOTE: This disables the abillity to build amd64 bootstraps with
current config.
---
 build/f_isoroot/f_repobuild/fuel_bootstrap_cli.yaml | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/build/f_isoroot/f_repobuild/fuel_bootstrap_cli.yaml b/build/f_isoroot/f_repobuild/fuel_bootstrap_cli.yaml
index aa12c45..2e4843d 100644
--- a/build/f_isoroot/f_repobuild/fuel_bootstrap_cli.yaml
+++ b/build/f_isoroot/f_repobuild/fuel_bootstrap_cli.yaml
@@ -36,19 +36,19 @@
   repos:
     - name: ubuntu
       section: "main universe multiverse"
-      uri: "http://127.0.0.1:8080/mirrors/ubuntu"
+      uri: "http://ports.ubuntu.com/ubuntu-ports"
       priority:
       suite: trusty
       type: deb
     - name: ubuntu-updates
       section: "main universe multiverse"
-      uri: "http://127.0.0.1:8080/mirrors/ubuntu"
+      uri: "http://ports.ubuntu.com/ubuntu-ports"
       priority:
       suite: trusty-updates
       type: deb
     - name: ubuntu-security
       section: "main universe multiverse"
-      uri: "http://127.0.0.1:8080/mirrors/ubuntu"
+      uri: "http://ports.ubuntu.com/ubuntu-ports"
       priority:
       suite: trusty-security
       type: deb
@@ -60,2 +60,3 @@
   direct_repo_addresses:
+    - "ports.ubuntu.com"
     - "127.0.0.1"
